EssentialFeed/EssentialFeedTests/Image Comments/Presentation/ImageCommentsPresenterTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsPresenterTests: XCTestCase {
+	func test_title_isLocalized() {
+		XCTAssertEqual(ImageCommentsPresenter.title, localized("IMAGE_COMMENTS_VIEW_TITLE"))
+	}
+
+	func test_map_createsViewModel() {
+		let calendar = Calendar(identifier: .gregorian)
+		let locale = Locale(identifier: "en_US_POSIX")
+		let comments = [
+			ImageComment(id: UUID(),
+			             message: "First Message",
+			             createdAt: Date().adding(minutes: -5, calendar: calendar),
+			             username: "First UserName"),
+			ImageComment(id: UUID(),
+			             message: "Second Message",
+			             createdAt: Date().adding(days: -1, calendar: calendar),
+			             username: "Second UserName")
+		]
+
+		let viewModel = ImageCommentsPresenter.map(comments,
+		                                           date: Date(),
```

Comment:
By creating multiple instances of `Date` the test become fragile, and it may fail if the dates being compared are slightly different.
```suggestion
		let now = Date()
		let comments = [
			ImageComment(id: UUID(),
			             message: "First Message",
			             createdAt: now.adding(minutes: -5, calendar: calendar),
			             username: "First UserName"),
			ImageComment(id: UUID(),
			             message: "Second Message",
			             createdAt: now.adding(days: -1, calendar: calendar),
			             username: "Second UserName")
		]

		let viewModel = ImageCommentsPresenter.map(comments,
		                                           date: now,
```